pkg/agent/core/ngt/handler/grpc/handler_test.go

```diff
@@ -1177,48 +1184,105 @@ func Test_server_Insert(t *testing.T) {
 		}
 		return nil
 	}
-	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       args: args {
-		           ctx: nil,
-		           req: nil,
-		       },
-		       fields: fields {
-		           name: "",
-		           ip: "",
-		           ngt: nil,
-		           eg: nil,
-		           streamConcurrency: 0,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
 
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           args: args {
-		           ctx: nil,
-		           req: nil,
-		           },
-		           fields: fields {
-		           name: "",
-		           ip: "",
-		           ngt: nil,
-		           eg: nil,
-		           streamConcurrency: 0,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+	/*
+		- Equivalence Class Testing
+			- uint8, float32
+				- case 1: Insert vector success
+				- case 2: Insert vector with different dimension
+
+		- Boundary Value Testing
+			- uint8, float32
+				- case 1: Insert vector with 0 value success
+				- case 2: Insert vector with min value success
+				- case 3: Insert vector with max value success
+				- case 4: Insert with empty UUID fail
```

Comment:
:thinking: 
if you'd like to run both uint and float32, why do you use the same case number?
IMO, it should be not same case number

Reply:
I think that it will be easier for review, so I wrote in this format.
what do you think about it?
```suggestion
				- case 1.1 : Insert int vector with 0 value success
				- case 1.2: Insert float vector with 0 value success
				- case 2.1: Insert int vector with min value success
				....
```

Reply:
I think it would be better cuz there are possibilities test cases may vary by variable type.
IMO, it is easier to review for me, e.g.:
```
cast 1.x -> test cases when variable type is uint8
case 2.x -> test case when variable type is float32
```

just my personal preference.